lib/elixir/scripts/windows_installer/installer.nsi

```diff
@@ -93,15 +150,20 @@ Function FinishPageShow
     Abort
   ${EndIf}
 
-  ${NSD_CreateCheckbox} 0 0 195u 10u "&Add $INSTDIR\bin to %PATH%"
-  Pop $AddElixirToPathCheckbox
-  SendMessage $AddElixirToPathCheckbox ${BM_SETCHECK} ${BST_CHECKED} 0
+  ; we add to PATH using erlang, so there must be an OTP installed to do so.
+  ${If} "$OTPPath" != ""
+    ${NSD_CreateCheckbox} 0 0 195u 10u "&Add $INSTDIR\bin to %PATH%"
+    Pop $AddElixirToPathCheckbox
+    SendMessage $AddElixirToPathCheckbox ${BM_SETCHECK} ${BST_CHECKED} 0
 
-  EnumRegKey $0 HKLM "SOFTWARE\WOW6432NODE\Ericsson\Erlang" 0
-  ReadRegStr $0 HKLM "SOFTWARE\WOW6432NODE\Ericsson\Erlang\$0" ""
-  ${NSD_CreateCheckbox} 0 20u 195u 10u "&Add $0\bin to %PATH%"
-  Pop $AddOTPToPathCheckbox
-  SendMessage $AddOTPToPathCheckbox ${BM_SETCHECK} ${BST_CHECKED} 0
+    ${NSD_CreateCheckbox} 0 20u 195u 10u "&Add $OTPPath\bin to %PATH%"
+    Pop $AddOTPToPathCheckbox
+    ${If} $OTPVerified == "true"
+      SendMessage $AddOTPToPathCheckbox ${BM_SETCHECK} ${BST_CHECKED} 0
+    ${EndIf}
```

Comment:
I've made it so that if we detected an OTP version we show the checkbox to add OTP to path. If the installed OTP release matches the installer, we check the checkbox. Otherwise it is unchecked. WDYT?

<img width="493" alt="image" src="https://github.com/elixir-lang/elixir/assets/76071/992e9231-3a2e-4194-9a03-eda4e5e167b1">

Reply:
maybe it was silly ie if we show the checkbox it is pre-checked.

Reply:
yeah never mind. If we found an OTP we show these checkboxes and they are always pre-checked. sorry about the noise.